src/components/pages/rules/flyouts/ruleEditor/ruleEditor.js

```diff
@@ -352,17 +356,19 @@ export class RuleEditor extends LinkedComponent {
             </FormGroup>
           </Section.Content>
         </Section.Container>
-        <SummarySection>
+        <SummarySection indicator={isPending} indicatorSize="large" indicatorPattern="bar">
```

Comment:
You don't need to do this. You just need to modify the the css a bit. 

In the SummarySection component update the container class to have `flex-flow: row wrap;` Then the following will work.

```
<SummarySection>
  <Indicator size="medium" pattern="bar" />
  <SummaryCount>{devicesAffected}</SummaryCount>
  <SectionDesc>{t('rules.flyouts.ruleEditor.devicesAffected')}</SectionDesc>
</SummarySection>
```